src/commands/validate.ts

```diff
@@ -30,5 +30,9 @@ export default class Validate extends Command {
     }
 
     await validate(this, specFile, flags);
+    if(await validate(this, specFile, flags) == ValidationStatus.Invalid) {
```

Comment:
actually why validate twice?

also `this.exit(0)` is not really necessary

Reply:
@derberg  changed it